webpack/assets/javascripts/jquery.ui.custom_spinners.js

```diff
@@ -135,12 +147,15 @@ export function initCounter() {
   });
 }
 
-export function initByte() {
-  $('input.byte_spinner').each(function() {
+export function initByte(selection = null) {
+  $('input.byte_spinner', selection).each(function() {
```

Comment:
I believe that with this:
```suggestion
  $('input.byte_spinner, selection).not('.form_template input.byte_spinner').each(function() {
    // ....
```
we could leave out the if.

Reply:
Nice, I just added it.
eslint thinks the lines are too long, so it resulted in some code reformatting.